### DISCUSSION TITLE
Possibly use drag-and-drop to open files.

### DISCUSSION THREAD

**Discussion by justinmcgrath** [akiyosi/goneovim#283] [Ideas]
In qt-neovim dragging and dropping a file onto the window opens the file. Goneovim doesn't appear to do anything when you do that. I find that it's a nice feature that I miss with this program. Any possibility of adding it?

**Comment by akiyosi** [comment]
@justinmcgrath 
Hi,
goneovim supports drag and drop, but it doesn't work well in your environment?

**Comment by akiyosi** [comment]
@justinmcgrath 
v0.4.12 may not work well with drag and drop. Please try the latest version below.

https://github.com/akiyosi/goneovim/actions/runs/1410379808

**Comment by justinmcgrath** [comment]
Using v0.4.12 dragging and dropping on to the window seems to have no effect. Using the version at the link provided (Goneovim-latest-win64-5.14.1), dragging and dropping freezes the program. The program doesn't seem to work in general though. It doesn't update the display until the window loses focus.

I'm using Windows 10, Neovim v0.5.0.

**Comment by justinmcgrath** [comment]
Changing the "Start in" directory for the shortcut I used to pass the "--nvim" option has fixed it and drag and drop works. Thank you.

**Reply by justinmcgrath** [reply]
Well, I take it back. It did work, but now it doesn't. I don't really know what's going on.

**Reply by akiyosi** [reply]
@justinmcgrath 
neovim 0.5.0 contains a bug that may cause it to crash. Could you please try to use the latest neovim head?

https://github.com/neovim/neovim/releases/tag/nightly

**Comment by justinmcgrath** [comment]
That fixes the crashing, but the display still doesn't show properly. It seems to have something to do with resizing the window. When it's first opened, it works fine. Making the window smaller always seems fine. Making it larger sometimes, but not always, causes it to stop updating as I type. It updates when it loses focus. Once it's in that state, it stays there until restarted. Resizing to a size that worked previously doesn't fix it.

Neovim 0.5.1, Goneovim-latest-win64-5.14.1. I start GoNeovim with a taskbar shortcut with this target `C:\Users\Justin\Programs\Goneovim-latest-win64-5.14.1\goneovim.exe --nvim="%HOMEPATH%\Programs\Neovim\bin\nvim.exe"`

Bypassing custom configurations with the following target still has the problem: `C:\Users\Justin\Programs\Goneovim-latest-win64-5.14.1\goneovim.exe --nvim="%HOMEPATH%\Programs\Neovim\bin\nvim.exe" - u NONE`

**Reply by akiyosi** [reply]
@justinmcgrath 
I'd like to take a moment to summarize what has happened so far.
Are the following perceptions correct?
* The drag-and-drop feature, which was originally requested, works in the latest version.
* The crash problem was solved in Neovim v0.5.1.
* There is a display problem between the latest Neovim v0.5.1 and the latest version of Goneovim.

Also, does the third problem above relate to the drag-on-drop feature? Or is it a general problem in normal use cases?

**Comment by justinmcgrath** [comment]
Those the points are correct.

The third problem is not related to drag-and-drop, but I only noticed it when upgrading Goneovim. However, I checked the version I had previously though (0.4.12) and it does the same thing. I think I never noticed before because it only seems to happen when I have an external monitor connected. On my laptop screen, everything works fine.

**Comment by akiyosi** [comment]
@justinmcgrath 
Thank you.
I had heard before about the behavior problems when using an external monitor. Unfortunately, I do not have an external monitor and have not been able to focus on that issue for improvement.
